### PR TITLE
feature/CF-1003 Change bootstrap script to use Terraform module

### PR DESCRIPTION
**Pull request by bartekrutkowski**

### CODE REVIEW

**Review by uxmankhan359 — DISMISSED** [review]

File: bootstrap/bootstrap.sh [hunk 1]
```diff
@@ -284,344 +290,61 @@ if ! gcloud auth application-default print-access-token >/dev/null 2>&1; then
...
-		log_error "Error occured during GCS bucket creation ... can't proceed"
-		exit 1
-	fi
 fi
-set_command_output_trap
 
-# Create IaC code from template
-TEMPLATE_BASEDIR=$(dirname "${BASH_SOURCE[0]}")
-TEMPLATE_DIR="${TEMPLATE_BASEDIR}/templates/"
+echo "Starting fitst stage terraform init." | fold -s -w 80
```
Comment (vmukhovatyi) [inline bootstrap/bootstrap.sh:320]: a small typo: fitst -> first

File: bootstrap/README.md [hunk 2]
```diff
@@ -1,14 +1,161 @@
...
+
+1. Terraform code from the `terraform` directory is being run using the executing user access to the [GCP IAM](https://cloud.google.com/iam/docs/groups-in-cloud-console) `manager_group` and does the following (in a nutshell):
+
+   - Grants the `manager_group` additional roles needed to perform the whole process
+   - Creates a service account intended for future use with Terraform IaC (also used in second stage)
+   - Creates a [GCP storage bucket](https://cloud.google.com/storage/docs/buckets) for Terraform [state file](https://developer.hashicorp.com/terraform/language/settings/backends/gcs)
+   - Generates Terraform code in output directory (by default: `iac-output`) to be used in second stage and further by the project users
+   - Sleeps for a given duration (by default: 5m) to give GCP time to [synchronize the IAM changes](https://cloud.google.com/iam/docs/access-change-propagation) to the groups, service accounts, etc.
+
+1. Terraform code from the `iac-output` (or a different one, if the `-o` parameter was used) is being run using the service account created in the first stage and does the following (in a nutshell):
```
Comment (vmukhovatyi) [inline bootstrap/README.md:57]: a small typo: 1. -> 2.

Comment (bartekrutkowski) [reply bootstrap/README.md:57]: This isn't a typo, right? This is Markdown syntax for the ordered lists. Can you check the document rendering with Markdown and confirm it's not a typo please?

File: bootstrap/bootstrap.sh (hunk 1 above)
Comment (bartekrutkowski) [reply bootstrap/bootstrap.sh:320]: Fixed.

File: bootstrap/README.md (hunk 2 above)
Comment (vmukhovatyi) [reply bootstrap/README.md:57]: yes you're correct.

**Review by vmukhovatyi — APPROVED** [review]
Approving once more after pre-commit changes were added

**Pull request merged**